HelpMyStreetFE/HelpMyStreetFE/src/js/shared/btn.js

```diff
@@ -2,6 +2,7 @@
 export function buttonLoad(btn) {        
     btn.width(btn.width());
     btn.height(btn.height());
+    btn.css('padding', "0");
```

Comment:
I love the ambition to fix this, but am not sure I can see a difference.  Is there a particular button you were looking at when testing this?

Reply:
Accept button is much better now :)

... although sadly the text in the Undo button that follows is now mis-aligned.  Boo!

Leave this one with me; I'll see if I can conjure up something that works everywhere.